systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -218,11 +219,7 @@ void testKafkaAndZookeeperScaleUpScaleDown() throws Exception {
         assertEquals(3, initialReplicas);
 
         // Create topic before scale up to ensure no partitions created on last broker (which will mess up scale down)
```

Comment:
Is this comment still valid? You seem to have a cluster with 3 topics and topic with 3 replicas?

Reply:
Removed